### PR TITLE
Add SSL cert,key,chain,disableCompressions,honorCipherOrder configuration

### PR DESCRIPTION
**Pull request by korniltsev**
Add configurations for these setters of tomcat's protocol 
```
SSLDisableCompression
SSLHonorCipherOrder
SSLCertificateFile
SSLCertificateKeyFile
SSLCertificateChainFile
```

### CODE REVIEW

File: spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java [hunk 1]
```diff
@@ -101,6 +101,31 @@
 	 */
 	private String protocol = "TLS";
 
+	/**
+	 * Disable SSL compression.
+	 */
+	private Boolean disableCompression;
```
Comment (wilkinsona) [inline spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java]: The `Ssl` configuration class is shared among all the embedded containers that we support (Jetty, Tomcat, and Undertow) which means that any configuration options that it offers have to be available across all three containers. Do you know if the 5 options you have proposed here are specific to Tomcat, or do Jetty and Undertow have equivalents for some/all of them?

File: spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java [hunk 2]
```diff
@@ -101,6 +101,31 @@
...
+
+	/**
+	 * Honor SSL cipher suite order.
+	 */
+	private String honorCipherOrder;
```
Comment (wilkinsona) [inline spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java]: While Tomcat uses a `String` for this it's coerced to a `boolean`. I think it'd be better if we made this more strongly typed and used a `Boolean`.

File: spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java (hunk 1 above)
Comment (korniltsev) [inline spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java]: Unfortunately I can't find some of these options in Jetty. I guess I'l just stick with a custom `EmbeddedServletContainerCustomizer` in my project.

**Comment by korniltsev** [comment]
Unfortunately I can't find some of these options in Jetty. I guess I'l just stick with a custom EmbeddedServletContainerCustomizer in my project.

**Pull request closed without merging**